packages/elements/line-height/src/defaults.ts

```diff
@@ -1,5 +1,5 @@
 export const KEY_LINE_HEIGHT = 'lineHeight';
 
-export const DEFAULT_LINE_HEIGHTS: number[] = [1, 1.2, 1.5, 2];
+export const DEFAULT_LINE_HEIGHTS = [1, 1.2, 1.5, 2, 3];
 
-export const DEFAULT_LINE_HEIGHT = 1.2;
+export const DEFAULT_LINE_HEIGHT = 1.5;
```

Comment:
This was 1.2 for a reason, while it is depending on the font, the usually the normal value is around for most fonts 1.2:
https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#values

Reply:
You're right, but what about editor standards?

https://ux.stackexchange.com/a/35275

What do you think?

Reply:
I think 1.5 is a nice default for reading, but generally 1.2-1.3 is more typical for editing so you can view more content at once.

Google Docs and MS Word default to 1.15 fwiw.

Reply:
Google Docs and MS Word have special typography which is more readable than in web.

I just gave a try with 1.2 and it feels quite bad.

![CleanShot 2021-10-30 at 03 56 06](https://user-images.githubusercontent.com/19695832/139516432-cf604bd3-fcdb-4a68-b755-3c3b9b77a481.png)


Tiptap and Notion both have 1.5 so let's stay on that default for now.

Reply:
Let's do that, it is configurable, so doesn't matter too much..